mounts.go

```diff
@@ -178,8 +178,8 @@ type MountEntry interface {
 }
 
 // FSContent represents file system content - it is based on the auto-generated
-// type Content which is json.RawMessage, which can be ArtifactContent or
-// URLContent concrete types. This is the interface which represents these
+// type Content which is json.RawMessage, which can be ArtifactContent, URLContent,
+// RawContent or Base64Content concrete types. This is the interface which represents these
```

Comment:
Can you wrap this comment at 80 chars? (In vim, go to line 180 (`:180<enter>`), then `V2jgw`).